backendservice/src/main/java/com/google/gpay/virtualqueue/backendservice/proto/GetShopResponse.java

```diff
@@ -0,0 +1,28 @@
+/*
+Copyright 2020 Google LLC
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+    https://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package com.google.gpay.virtualqueue.backendservice.proto;
+
+import java.util.concurrent.atomic.AtomicInteger;
+
+import com.google.gpay.virtualqueue.backendservice.model.Shop;
+
+import lombok.AllArgsConstructor;
+import lombok.Data;
+
+@Data
+@AllArgsConstructor
+public class GetShopResponse {
+    private Shop shop;
+    private AtomicInteger customersInQueue;
```

Comment:
Why do we want to use AtomicInteger here? Can we not use just 'long' primitive data type?

Reply:
To use java streams in finding the number of customers in queue, could not use long variable as a counter, as I read that it was not allowed inside a lambda expression. Please correct me if I am wrong.